bin/compile

```diff
@@ -11,4 +11,22 @@ mkdir -p $METABASE_DIR
 
 echo -n "-----> Downloading Metabase... from $METABASE_URL to $METABASE_JAR"
 curl -s --retry 3 -o $METABASE_JAR -L $METABASE_URL
+
+CSS=(styles embed)
```

Comment:
It's the only one we need to override the metabase embed. If we want more and also in the "main" metabase, we can uncomment the lines below